tic_tac_toe/game.py

```diff
@@ -9,7 +11,12 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
+    x = int(position[0])
```

Comment:
Your `_position_is_valid` method is already going to verify that these are `int`'s, so you don't need to cast them.